BlueprintUI/Sources/Measuring/SizeConstraint.swift

```diff
@@ -1,16 +1,44 @@
 import UIKit
 
+/// This property wrapper checks the value of `atMost` cases, and turns it into an
+/// `unconstrained` axis if the value equals `greatestFiniteMagnitude` or `isInfinite`.
+@propertyWrapper public struct UnconstrainedInfiteAxis: Equatable, Hashable {
```

Comment:
Nit: We may want to either move this to another file if we intend for it to be useful outside of `SizeConstraint`, or nest it within `extension SizeConstraint {` (which would also allow us to remove the `SizeConstraint.`s from `SizeConstraint.Axis` in this property wrapper).